project/Dependencies.scala

```diff
@@ -114,6 +114,8 @@ object Dependencies {
   val slf4jApi: ModuleID = "org.slf4j" % "slf4j-api" % slf4jVersion
   val slf4jSimple: ModuleID = "org.slf4j" % "slf4j-simple" % slf4jVersion
 
+  val okio: ModuleID = "com.squareup.okio" % "okio" % "3.4.0"
```

Comment:
You'll need to exclude `okio` from workbench-libs, since its a transitive dependency of that. Try running `sbt dependencyBrowseTree` to see which wb-libs artifacts need the exclusion

Reply:
Ah yeah I forgot about that thanks.